### PR TITLE
Create files to allow us to fix the listing on Artifacthub

### PR DESCRIPTION
**Pull request by jkowall**
#### What this PR does
Adds a file for us to identify this repo as the official chart. This will allow me to correct the listing as official and fix a security gap. I added myself to the Chart and new artifacthub-repo.yml. 

I have also bumped the version for Jaeger to current and changed the version of the chart.

#### Which issue this PR fixes
https://github.com/jaegertracing/jaeger/issues/5363

#### Checklist

- [X] [DCO](https://github.com/jaegertracing/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Commits are [GPG signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (`[jaeger]` or `[jaeger-operator]`)
- [X] README.md has been updated to match version/contain new values

### CODE REVIEW

File: charts/jaeger/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: 1.53.0
+appVersion: 1.56.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
 type: application
-version: 3.0.0
+version: 3.0.1
```
Comment (yurishkuro) [inline charts/jaeger/Chart.yaml:6]: maybe avoid these changes with an otherwise administrative PR

**Comment by yurishkuro** [comment]
it seems this chart is already listed https://artifacthub.io/packages/helm/jaegertracing/jaeger, but it's missing all the "official" badges. 

image

In contrast, Prometheus:

image

File: charts/jaeger/Chart.yaml (hunk 1 above)
Comment (jkowall) [reply charts/jaeger/Chart.yaml:6]: The chart version must be bumped. Hence makes sense to bump jaeger too.

Comment (yurishkuro) [reply charts/jaeger/Chart.yaml:6]: yes, but it should be a separate PR dedicated to bumping version

Comment (alex1989hu) [reply charts/jaeger/Chart.yaml:6]: @jkowall 
Newer Jaeger requires RBAC change as well. Please take a look https://github.com/jaegertracing/helm-charts/issues/549

Comment (pavelnikolov) [reply charts/jaeger/Chart.yaml:6]: The chart version, indeed, needs to be bumped with every change. However, the Jaeger version bumped needs to be a separate PR.

Comment (jkowall) [reply charts/jaeger/Chart.yaml:6]: Reverted

**Review by yurishkuro — APPROVED** [review]

**Review by pavelnikolov — APPROVED** [review]

**Comment by jkowall** [comment]
Will reopen now that my signing is fixed

**Pull request closed without merging**